frontend/src/citizen-frontend/applications/editor/validations.ts

```diff
@@ -92,12 +92,16 @@ export const isValidDecisionStartDate = (
   startDate: string,
   type: DecisionType
 ): boolean => {
-  return date !== null
-    ? date.isEqualOrAfter(LocalDate.parseFi(startDate)) &&
-        date.isEqualOrBefore(
-          maxDecisionStartDate(LocalDate.parseFi(startDate), type)
-        )
-    : false
+  try {
+    return date !== null
+      ? date.isEqualOrAfter(LocalDate.parseFi(startDate)) &&
+          date.isEqualOrBefore(
+            maxDecisionStartDate(LocalDate.parseFi(startDate), type)
+          )
+      : false
+  } catch (e) {
+    return false
+  }
```

Comment:
Don't probably want to hide all possible exceptions from all the functions + just parsing `startDate` once would be preferable. My suggestion (comment-coding, cleanup needed):

```suggestion
  let parsedDate: LocalDate
  try {
    parsedDate = LocalDate.parseFi(startDate)
  } catch (e) {
    if (e instanceof RangeError) {
      return false
    }
    // Unexpected
    throw e
  }
  return date !== null
    ? date.isEqualOrAfter(parsedDate) &&
        date.isEqualOrBefore(maxDecisionStartDate(parsedDate, type))
    : false
```

Reply:
Thanks, very nice suggestion 💎.